skawa_components/pubspec.yaml

```diff
@@ -27,7 +27,10 @@ dev_dependencies:
   mockito: ^4.0.0
   test: ^1.6.2
   angular_test: ^3.0.0
-  pageloader: ^3.2.0
+  pageloader:
```

Comment:
why this line change is necessary?

Reply:
I changed this line because, at first I wanted to test the change,  and build before the tests failed, this change  solved it. Later found out that this feature couldn't be be tested, but I kept this because I didn't want to other face with the same issue.